modules/tree/src/main/tree.scala

```diff
@@ -118,7 +403,9 @@ object Node:
   object Comment:
     opaque type Id = String
     object Id extends OpaqueString[Id]:
-      def make = Id(ThreadLocalRandom nextString 4)
+      // TODO: REVERT THIS
```

Comment:
Temporary makes this returns constant for testing purpose: https://github.com/lichess-org/lila/pull/12696/files#diff-0513e1a697b9f01264af1ec546af0385d0a737bd75c086007c80b13418099bebR159